src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -73,6 +76,8 @@ typedef struct WalkerState
 } WalkerState;
 
 bool EnableRouterExecution = true;
+static uint32 varEquivalenceId = 1;
```

Comment:
Eventually this would overflow. Should we reset this when we start planning a query ?